src/types/Storage/Storage.spec.js

```diff
@@ -28,6 +29,12 @@ describe('Storage - constructor', () => {
     storage.stopWorker();
   });
   it('should handle bad adapter', async () => {
+    if(process.browser){
```

Comment:
Let's use `this.skip` with comment as well

Reply:
Same thing than other, why to remove something that actually test in 99.9% of the case if just skipping it on the browser solves our issue ?

Reply:
I meant:
```js
if (process.browser){
  this.skip('doesn\'t work in browser')
  return;
}
```